### DISCUSSION TITLE
Audio Compression

### DISCUSSION THREAD

**Discussion by matthewunmuted** [savonet/liquidsoap#2695] [User Support]
Hi there,

We have a client who is reporting that they have found their audio is being compressed more than their backup server when streaming using our services through MediaCP, using LiquidSoap.

![compre](https://user-images.githubusercontent.com/111743235/196378134-214b3da8-4416-49d3-9235-dee10f4888ec.jpg)

Attached is the image they sent us showing the difference in compression when using their backup server and our server.

Is this something that we are able to change for the client, or can this only be done on dedicated servers?

Thanks in advance.

**Comment by toots** [comment]
Thanks for reporting! Do you know the encoder used in each case?

**Comment by matthewunmuted** [comment]
Hi toots - I've asked the client, and will get back to you as soon as possible. Thanks
